tests/e2e/parallel/main_test.go

```diff
@@ -731,6 +671,242 @@ func TestSingleScanTimestamps(t *testing.T) {
+		t.Fatal(err)
+	}
+}
+
+func TestScanTailoredProfileExtendsDeprecated(t *testing.T) {
+	t.Parallel()
+	f := framework.Global
+
+	pbName := framework.GetObjNameFromTest(t)
+	baselineImage := fmt.Sprintf("%s:%s", brokenContentImagePath, "deprecated_profile")
+	pb, err := f.SetUpProfileBundle(pbName, baselineImage, framework.OcpContentFile)
+	if err != nil {
+		t.Fatalf("failed to create ProfileBundle: %s", err)
+	}
+	// This should get cleaned up at the end of the test
+	defer f.Client.Delete(context.TODO(), pb)
+
+	if err := f.WaitForProfileBundleStatus(pbName, compv1alpha1.DataStreamValid); err != nil {
+		t.Fatalf("failed waiting for the ProfileBundle to become available: %s", err)
+	}
+
+	tpName := "test-tailored-profile-extends-deprecated"
+	tp := &compv1alpha1.TailoredProfile{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      tpName,
+			Namespace: f.OperatorNamespace,
+		},
+		Spec: compv1alpha1.TailoredProfileSpec{
+			Extends:     "ocp4-cis-1-4",
+			Title:       "TestScanTailoredProfileExtendsDeprecated",
+			Description: "TestScanTailoredProfileExtendsDeprecated",
+			EnableRules: []compv1alpha1.RuleReferenceSpec{
+				{
+					Name:      "no-netrc-files",
+					Rationale: "Test for platform profile tailoring",
+				},
+			},
+		},
+	}
+	createTPErr := f.Client.Create(context.TODO(), tp, nil)
+	if createTPErr != nil {
+		t.Fatal(createTPErr)
+	}
+	defer f.Client.Delete(context.TODO(), tp)
+
+	scanName := framework.GetObjNameFromTest(t)
+	testScan := &compv1alpha1.ComplianceScan{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      scanName,
+			Namespace: f.OperatorNamespace,
+		},
+		Spec: compv1alpha1.ComplianceScanSpec{
+			Profile:      "xccdf_compliance.openshift.io_profile_test-tailored-profile-extends-deprecated",
+			Content:      framework.OcpContentFile,
+			ContentImage: baselineImage,
+			ComplianceScanSettings: compv1alpha1.ComplianceScanSettings{
+				Debug: true,
+			},
+		},
+	}
+	// use Context's create helper to create the object and add a cleanup function for the new object
+	if err = f.Client.Create(context.TODO(), testScan, nil); err != nil {
+		t.Fatalf("failed to create scan %s: %s", scanName, err)
+	}
+	defer f.Client.Delete(context.TODO(), testScan)
+
+	// The profile deprecation warning is sent out during Pending phase
+	if err = f.WaitForScanStatus(f.OperatorNamespace, scanName, compv1alpha1.PhasePending); err != nil {
+		t.Fatal(err)
+	}
+
+	if err = f.WaitForProfileDeprecatedWarning(t, scanName, tpName); err != nil {
+		t.Fatal(err)
+	}
+
+	if err = f.WaitForScanStatus(f.OperatorNamespace, scanName, compv1alpha1.PhaseDone); err != nil {
+		t.Fatal(err)
+	}
+}
+
+func TestScanTailoredProfileIsDeprecated(t *testing.T) {
+	t.Parallel()
+	f := framework.Global
+
+	tpName := "test-tailored-profile-is-deprecated"
+	tp := &compv1alpha1.TailoredProfile{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      tpName,
+			Namespace: f.OperatorNamespace,
+			Annotations: map[string]string{
```

Comment:
Ok - intersting, so this will let cluster administrators signal to users (e.g., other cluster admins?) that a tailored profile is deprecated.

Reply:
Correct, cluster admins have one more tool to manage `TailoredProfiles`